README.md

```diff
@@ -60,3 +60,26 @@ tiatoolbox --help
       -h, --help            show this help message and exit
       --version             show program`s version number and exit
       --verbose VERBOSE
+
+
+License
+=======
+
+The source code TIA Toolbox (tiatoolbox) as hosted on GitHub is released under the [GNU General Public License (Version 3)].
+
+The full text of the licence is included in [LICENSE.md](LICENSE.md).
+
+[GNU General Public License (Version 3)]: https://www.gnu.org/licenses/gpl-3.0.html
+
+
+Auxiliary Files
+---------------
+
+Auxiliary files which are downloaded from sources outside of this repository, such as pre-trained model weights, are provided under the [Creative Commons Attribution-NonCommercial-ShareAlike Version 4 (CC BY-NC-SA 4.0) license](https://creativecommons.org/licenses/by-nc-sa/4.0/).
```

Comment:
Auxiliary files, such as pre-trained model weights, are provided under the  [Creative Commons Attribution-NonCommercial-ShareAlike Version 4 (CC BY-NC-SA 4.0) license](https://creativecommons.org/licenses/by-nc-sa/4.0/).

Reply:
I think we need to be clear which files are which. E.g. state files on GitHub vs not or have a header in each file. Otherwise, it is ambiguous as to which files are under which licence.

Reply:
Can we say "Auxiliary files, such as pre-trained model weights downloaded using default links in the repo, are provided under the ..."

Reply:
Auxiliary files, such as pre-trained model weights downloaded from [tiatoolbox](https://warwick.ac.uk/fac/sci/dcs/research/tia/tiatoolbox), are provided under the ...

Reply:
This should be updated now. I included the full link in brackets because I though it was more clear.